composer.json

```diff
@@ -61,7 +61,7 @@
             "common": "common.php",
             "console": "console.php",
             "events-console": "events-console.php",
-            "providers-web": "providers-web.php"
+            "providers": "providers-web.php"
```

Comment:
File should be renamed as well, I think.

Reply:
The file is renamed, see below

Reply:
Yes but in this line it is still `"providers-web.php"`

Reply:
Fixed